silk/models.py

```diff
@@ -58,7 +57,6 @@ def __init__(self, d):
 
 
 class Request(models.Model):
-    id = CharField(max_length=36, default=uuid4, primary_key=True)
```

Comment:
The default id field for a django model is going to be a BigAutoField which is an integer field.  Removing this line would change a `Request` model from an integer field to a charfield, which would probably break a lot of dependencies or require wiping a lot of data.  

 - https://docs.djangoproject.com/en/4.1/topics/db/models/#automatic-primary-key-fields